src/nvim/drawline.c

```diff
@@ -2629,10 +2638,10 @@ int win_line(win_T *wp, linenr_T lnum, int startrow, int endrow, int col_rows, s
     // 'list' is set.
     if (wp->w_p_lcs_chars.ext != NUL
         && wp->w_p_list
-        && !wp->w_p_wrap
+        && !is_wrapped
```

Comment:
This should also still use `w_p_wrap`

Reply:
Actually I think the `has_foldtext` below should also be changed back to `has_fold` as showing 'listchars' `extends` over 'fillchars' `fold` looks kind of ugly.